internal/aws/metrics/metric_calculator_test.go

```diff
@@ -130,14 +139,14 @@ func TestMapWithExpiryCleanup(t *testing.T) {
 	assert.Equal(t, 1, store.Size())
 	store.Unlock()
 
-	time.Sleep(time.Second + time.Millisecond)
+	time.Sleep(time.Millisecond * 2)
```

Comment:
Is this sleep still needed?

Reply:
Is there ever a chance that `time.Now()` can return the same value? If so that would put a requirement for the sleep.